helpers_test.go

```diff
@@ -0,0 +1,31 @@
+package wampproto_cli_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	main "github.com/xconnio/wampproto-cli"
```

Comment:
```suggestion
	main "github.com/xconnio/wampproto-cli"
```
use the package name, not main